2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,11 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
+  return {
+    name: recipe['name'],
+    items: recipe['ingredients'].filter(ingredient => {
+      return !pantry.fridgeContents.includes(ingredient) && !pantry.cupboardContents.includes(ingredient);})
```

Comment:
this arrow can be implemented inline, without the curly brackets and return statement, can you amend it?